101-webapp-basic-windows/azuredeploy.parameters.us.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
remove this file

Reply:
The Gov build fails unless I use the location of 'usgovtexas'. Since you asked to remove this, I put a workaround to limit the test to the Azure Public. Let me know what you recommend.